openstack/neutron/Chart.yaml

```diff
@@ -29,3 +29,6 @@ dependencies:
     repository: https://charts.eu-de-2.cloud.sap
     version: 1.2.29
```

Comment:
We need to bump redis to at least `1.3.5` to make it support linkerd.

Reply:
Updated it now,  thanks !